web/js/map/map.js

```diff
@@ -4,6 +4,10 @@ import loIsUndefined from 'lodash/isUndefined';
 import loFind from 'lodash/find';
 import OlGeomPolygon from 'ol/geom/polygon';
 
+export const CRS_WGS_84 = 'EPSG:4326';
```

Comment:
Where was this constant defined before? I suspect that maybe the bug we're seeing is related to this constant.

Reply:
here within`self`: https://github.com/nasa-gibs/worldview/blob/master/web/js/map/wv.map.js#L8